wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/podcasts/PodcastsScreen.kt

```diff
@@ -52,15 +56,34 @@ fun PodcastsScreen(
         item {
             ScreenHeaderChip(if (uiState.folder == null) stringResource(LR.string.podcasts) else uiState.folder.name)
         }
-        items(items = uiState.items, key = { item -> item.uuid }) { item ->
-            when (item) {
-                is FolderItem.Podcast -> {
-                    PodcastChip(podcast = item, onClick = navigateToPodcast)
-                }
-                is FolderItem.Folder -> {
-                    FolderChip(folderItem = item, onClick = navigateToFolder)
+        if (uiState.items.isNotEmpty()) {
```

Comment:
This is good, it might be slightly better if we move this logic to the ViewModel and handle `Empty`, `Loaded` states returned from it in the `PodcastScreen`. We can convert `PodcastsViewModel` `UiState` to a sealed class as below:

```
sealed class UiState {
    object Empty : UiState()
    data class Loaded(
         val folder: Folder? = null,
         val items: List<FolderItem> = emptyList()
    ) : UiState()
}
```